user/multi-platform-login.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Multi-Platform Login
+layout: en
+
+---
+# Multiple Logins on Travis CI
+
+You can now log into Travis CI using several accounts from our supported platform providers and even switch between them.
+
+> For Travis CI being able to recognize all your accounts, you have to log in on the provider website and then add the account on [Travis CI](travis-ci.com/signin).
+
+## Adding Accounts
+
+You can add more accounts to Travis CI by using either the Sign-in page or the user menu.
+
+### Adding accounts using the Sign-in page
+
+To add more Bitbucket accounts:
+
+1. Go to the Bitbucket login site.
+2. Re-login on Bitbucket with the desired account.
+3. Go to the Travis CI Sign-in page.
+4. Click **Sign In with Bitbucket** on the sign-in page. Your account is stored and you can now switch between the different accounts in Travis CI.
+
+### Adding accounts using the user menu
+
+To add more accounts:
+
+1. Click on your user icon in the top right corner of [Travis CI](travis-ci.com).
+2. Click the **Add an account** link at the bottom of the menu. The Sign-in page appears displaying all your accounts.
```

Comment:
Should we also add here `re-login on Bitbucket with the desired account`?

Reply:
I don't think we need to mention this here as it is in the Note right on top